src/backend/distributed/operations/shard_split.c

```diff
@@ -545,13 +553,15 @@ SplitShard(SplitMode splitMode,
  * sourceColocatedShardIntervalList : Source shard group to be split.
  * shardSplitPointsList             : Split Points list for the source 'shardInterval'.
  * workersForPlacementList          : Placement list corresponding to split children.
+ * needsSeparateNodeForShardList    : Whether each split children needs a separate node.
  */
```

Comment:
nit; apparently we miss documentation for `distributionColumnOverrides`, do you happen to know what it is?

mmight be worth to do in a separate PR though.

Reply:
>  do you happen to know what it is?

Don't really know what it is about :/

> mmight be worth to do in a separate PR though.

👍